src/pretix/plugins/paypal/payment.py

```diff
@@ -115,7 +184,13 @@ def checkout_prepare(self, request, cart):
                         "currency": request.event.currency,
                         "total": self.format_price(cart['total'])
                     },
-                    "description": __('Event tickets for {event}').format(event=request.event.name)
+                    "description": __('Event tickets for {event}').format(event=request.event.name),
+                    "payee": {
+                        "email": "" or request.event.settings.payment_paypal_connect_user_id,
```

Comment:
``"" or B`` does not make sense, it will always return ``B``. Did you mean ``… or ""``?

Reply:
Again, agreed: Will flip those two around.